The-Politico/artisan
# Actions based on 10/3 tickets

**Done**:

- [Create illustration](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/CreateIllustration) 
- [Open Illustration](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/OpenIllustration)
- [Duplicate Project](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/DuplicateProject)
- [Delete Illustration](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/DeleteIllustration)
- [Generate](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/Generate)
- [CreateProject](https://github.com/The-Politico/artisan/tree/actions-by-rashida/src/actions/CreateProject)
- ~[Share](https://www.notion.so/the-politico/Artisan-Action-Share-c244f8c6b01f42039ea55de3ec826125)~ (This is just going to be a regular button / link we click with the share URL)
- [Preview](https://www.notion.so/the-politico/Artisan-Action-Preview-f3703b56c26b4e0db78e9613c13ad505)

I've also replaced the ai2html and config files in src-tauri with [the updated files we created a few months ago](https://github.com/The-Politico/test_ai2html-2022/blob/main/ai2html.js). I've adjusted the output directory to match expectations of Artisan. 

# To test this suite of actions: 

+ Run `npm run tauri dev`
+ Artisan expects your project folder to be something like: `abriz/Documents/Artisan/Projects/` so make sure that those folders exist. 

You should be able to click through in the order of the displayed buttons. To check that actions are working, you'll need to see what's happening in your project folder. For now, the actions have been supplied with input such as "proj-1" and "test-illo" from ActionTestingView. Eventually, those values will need to be supplied by user input.
